client/settings/fraud-protection/interfaces.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { Dispatch, SetStateAction } from 'react';
+
+export interface FraudPreventionSetting {
+	block: boolean;
+	enabled: boolean;
+	min_items?: string | number | null;
```

Comment:
Will it be a good idea to split this into multiple interfaces? I think, for example, the purchase price threshold can initialize without `min_amount` in the provided setting object this way.

Reply:
I think it could be a good idea because it could make it easier to to understand and maintain the code. That said, we do have some pending changes to make to the advanced settings based on feedback from the demo. So I don't know if we should do it now or as part of those pending changes. What do you think @eduardoumpierre?

If we move forward with it now, I can split it out into the following five interfaces:
- International IP Address
- IP Address Mismatch
- Address Mismatch
- Purchase Price Threshold
- Order Items Threshold

Reply:
No need for five interfaces, just a standard one, and two additional ones for the rules that require extra parameters (purchase price and order items), so we can use a combined type, like described here: https://ultimatecourses.com/blog/use-intersection-types-to-combine-types-in-typescript

Reply:
Okay we've addressed this in [e771f99](https://github.com/Automattic/woocommerce-payments/pull/6032/commits/e771f99dd0ff3557e0eb947de2def477714faed9). I think you should now be good to re-review and test. 

Let us know if you have any further feedback.